includes/utils.php

```diff
@@ -755,4 +755,16 @@ public static function print_wp_kses_extended( $string, array $tags ) {
 
 		echo wp_kses( $string, $allowed_html );
 	}
+
+	public static function is_elementor_path( $path ) {
```

Comment:
Should also support 3rd party plugins (maybe add filter)

Reply:
https://github.com/elementor/elementor/pull/12246/commits/6ba8334ec1f6c509811e161729fa3be04fd13f57